docs/_posts/bunyamin-polat/2024-01-17-snomed_findings_resolver_pipeline_en.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: model
+title: Pipeline for Snomed Concept, CT version
```

Comment:
@bunyamin-polat ~Pipeline for Snomed Concept, CT version`~ : Pipeline for Snomed Concept, Findings Version